docs/logs/fields.md

```diff
@@ -4,4 +4,68 @@ description: Sematext Logs App looks for special fields in logs, namely os.host,
 Every log event shipped to Sematext Logs has its structure - it is divided into fields. Each field has a [type](/logs/field-types/), for example [string](/logs/field-types/#string), [date](/logs/field-types/#date), or [integer](/logs/field-types/#integerlong). It can even be an object holding structured data.  We do everything we can to ensure that log event field types are inferred correctly. However, you may also want to set field types explicitly. This can be done using the Field Editor accessible via a Logs App settings or using the [Templates and Mappings](/logs/mappings-templates) APIs.
 
 ## Common Schema
-Fields in log events are also referred to as Tags in Sematext.  They are used for searching and filtering, but also for pivoting from Logs to other observability data in Sematext, such as performance metrics in [Monitoring](../monitoring).  The [Common Schema for Logs](../tags/common-schema/#logs-tags) lists special fields and their meaning.
+Fields in log events are also referred to as Tags in Sematext. They are used for searching and filtering, but also for pivoting from Logs to other observability data in Sematext, such as performance metrics in [Monitoring](../monitoring).  The [Common Schema for Logs](../tags/common-schema/#logs-tags) lists special fields and their meaning.
+
+## Fields Structure
+
+The structure of your data is dictated by the data shipped to Sematext Logs as log events. There are two places where you can easily see your data structure - the fields and filters section of your Logs table and the [Field Editor](/logs/fields/#field-editor):
```

Comment:
The structure of your data is dictated by the data shipped to Sematext Logs as log events.
=> 
The structure of your logs - their fields and types - is automatically created when you ship your logs to a Logs App in Sematext.

There are two places where you can easily see your data structure => 
There are two places where you can easily see your fields 

> the fields and filters section of your Logs table

We don't call it fields & filters in the UI, so maybe we need to mark up the first screenshot below?
Also, F&F may technically not be a part of LT, or are they?

Reply:
Done and you are right F&F is technically not part of the LT. I rephrased and split it, so it should be easy to see what the docs are talking about.